components/post_view/post_image.jsx

```diff
@@ -41,6 +44,7 @@ export default class PostImageEmbed extends React.PureComponent {
             loaded: false,
             errored: false
         };
+        this.imagesAreProxied = global.window.mm_config.HasImageProxy === 'true';
```

Comment:
I'd prefer to do this using a connector (index.js) and passing it in as a regular prop but not a big enough deal to block merging